### PR TITLE
Fixes for spots in starfish.display

### PR DESCRIPTION
**Pull request by ambrosejcarr**
* Add ability of starfish.display to adjust z-size of spots to account for anisotropy
* Enable n-d spots in starfish.display
* Change default display colors to gray colormap + red spots

### CODE REVIEW

File: starfish/_display.py [hunk 1]
```diff
@@ -246,9 +253,12 @@ def display(
             return viewer
 
         coords = coords[mask, :]
-        sizes = sizes[mask]
+        sizes = sizes[mask, :]
```
Comment (kne42) [inline starfish/_display.py]: shouldn't this be equivalent to `sizes = sizes[mask]`?

Comment (ambrosejcarr) [reply starfish/_display.py]: In this regime, sizes is a (spots, 5) matrix, just like coords.  Can you point me at `napari` tests? I'd like to ideally add some to test various edge cases (2d, 3d, pseudo-3d where `z` is a singleton)

Comment (kne42) [reply starfish/_display.py]: I mean those two lines behave exactly the same, assuming `sizes` is of the right shape; just don't see the need for the explicit `, :`. There are no napari tests for high-level things like that other than creating a script and looking at it yourself so that's just what I would do.
